gui/src/app/mod.rs

```diff
@@ -206,3 +216,75 @@ impl App {
         self.state.view(&self.cache).map(Message::View)
     }
 }
+
+/// Possible errors when starting bitcoind.
+#[derive(PartialEq, Eq, Debug, Clone)]
```

Comment:
please move all the code from 221 to the end to a new module in `src/bitcoind.rs`, part of the code is also used by loader and we want to keep loader and app code separated.

Reply:
I've done this in https://github.com/wizardsardine/liana/pull/592/commits/36446048939d4006650b2535fb86256a74603b09 and https://github.com/wizardsardine/liana/pull/592/commits/9f0b92149f609fdc58f4f5ed75461544cad627d5 (forgot to add the new file in first commit :blush:).

Reply:
@jp1ac4 feel free to squash your commits